benchmark/dataset_benchmark.cpp

```diff
@@ -233,42 +233,21 @@ BENCHMARK(BM_Dataset_cache_blocking_no_slicing)
     ->RangeMultiplier(2)
     ->Range(2 << 9, 2 << 14);
 
-Dataset makeBeamline(const gsl::index nComp, const gsl::index nDet) {
-  Dataset d;
+Dataset makeBeamline(const gsl::index nDet) {
+  // TODO The created beamline is currently very incomplete so the full cost
```

Comment:
Why is `makeBeamline` made to be less complete now than it was before?

Reply:
There were a lot of tags with "fake" types which I had used for some benchmarking. Currently they were just contributing to a lot of noise (many tags, unclear what they were for), so I decided to remove them to avoid confusion.

Reply:
Those fake tags had a 1:1 correspondence to things we had in `Beamline::DetectorInfo` and `Beamline::ComponentInfo`, but the correct underlying types were unavailable and there was no usable functionality for them.